std/typecons.d

```diff
@@ -2999,24 +2999,33 @@ Returns:
     assert(i == 5);
 }
 
+    deprecated("Implicit conversion with `alias Nullable.get this` is deprecated. Please use `.get` explicitly.")
+    @property ref inout(T) get_() inout @safe pure nothrow
+    {
+        enum message = "Called `get' on null Nullable!" ~ T.stringof ~ ".";
+        assert(!isNull, message);
+        return _value.payload;
```

Comment:
Why not just forward to `get` ?

Reply:
Because I'm an idiot. Consider that fixed on tues first thing.